src/platform_impl/linux/window.rs

```diff
@@ -325,7 +335,7 @@ impl Window {
       preferred_theme: RefCell::new(preferred_theme),
     };
 
-    win.set_skip_taskbar(pl_attribs.skip_taskbar);
+    let _ = win.set_skip_taskbar(pl_attribs.skip_taskbar);
```

Comment:
I fixed it because the following warning message occurred.
<img width="750" alt="스크린샷 2024-10-02 오후 6 40 26" src="https://github.com/user-attachments/assets/fe7f1f5a-84aa-48c9-839e-541c19983033">